build-android/build.sh

```diff
@@ -29,7 +33,15 @@ if [ "${CLASSICAL}" == "1" ]; then
   popd
 
   mkdir -p /root/out/tools
-  cp bin/android_editor.apk /root/out/tools/
+  # Copy the generated Android editor binaries (apk & aab).
+  if [ ! -e bin/android_editor_builds/android_editor-release.apk ]; then
+    echo "No signed release builds, using debug builds instead."
+    cp bin/android_editor_builds/android_editor-debug.apk /root/out/tools/android_editor.apk
+    cp bin/android_editor_builds/android_editor-debug.aab /root/out/tools/android_editor.aab
```

Comment:
In my experience testing while I didn't have the environment variables properly sourced, when the signed release fail, what we get is actually:
```
android_editor-release-unsigned.apk
android_editor-release.aab
```
So this step fails.